src/main/java/com/github/thorbenkuck/netcom2/logging/CallerTraceSystemDefaultStyleLogging.java

```diff
@@ -25,13 +31,13 @@ public String getPrefix() {
 	 *
 	 * @return the class name of the class, calling any method to log something or null if no valid name is found
 	 */
-	String getCaller() {
+	private String getCaller() {
 		final StackTraceElement[] stackTraceElements = Thread.currentThread().getStackTrace();
 		for (StackTraceElement stackTraceElement : stackTraceElements) {
-			if (! stackTraceElement.getClassName().equals(CallerReflectionLogging.class.getName())
-					&& ! stackTraceElement.getClassName().equals(NetComLogging.class.getName())
-					&& ! stackTraceElement.getClassName().equals(CallerTraceSystemDefaultStyleLogging.class.getName())
-					&& ! stackTraceElement.getClassName().equals(SystemDefaultStyleLogging.class.getName())
+			if (!stackTraceElement.getClassName().equals(CallerReflectionLogging.class.getName())
```

Comment:
Ugh

Reply:
Yep, this is analysing the stacktrace and it is by no mean beautifull

Reply:
That, by the way, is the reason, why it should be used with great care.

Reply:
Really no way to make this better?